src/Tribe/Cache_Listener.php

```diff
@@ -90,6 +96,32 @@ public function update_last_save_post( $option_name, $old_value, $value ) {
 			}
 		}
 
+		/**
+		 * Run the caching functionality that is executed on saving tribe calendar options.
+		 *
+		 * @see 'updated_option'
+		 *
+		 * @param string $option_name    Name of the updated option.
+		 * @param mixed  $old_value The old option value.
+		 * @param mixed  $value     The new option value.
+		 */
+		public function update_last_updated_option( $option_name, $old_value, $value ) {
+			$triggers = [
+				'active_plugins',
+				'tribe_events_calendar_options',
+				'permalink_structure',
+				'rewrite_rules',
+				'start_of_week',
+				'sidebars_widgets',
+				'stylesheet',
+				'template',
+			];
+
+			if ( in_array( $option_name, $triggers, true ) ) {
```

Comment:
A suggestion on this one to remove the `n` iterations from `in_array` might be to convert the structure above to something like a hash.


```php
			$triggers = [
				'active_plugins' => true,
				'tribe_events_calendar_options' => true,
				'permalink_structure' => true,
				'rewrite_rules' => true,
				'start_of_week' => true,
				'sidebars_widgets' => true,
				'stylesheet' => true,
				'template' => true,
			];

```

With that you can just do a single lookup of the value on the triggers array like:


```php
if ( isset( $triggers[ $option_name ] ) ) {
}
```